library/.pre-commit-config.yaml

```
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.4.2
+    hooks:
+    -   id: insert-license
```

yamllint (1.35.1)

[warning] too many spaces after hyphen

(hyphens)